sections/dashboard/PortfolioChart/PortfolioChart.tsx

```diff
@@ -1,12 +1,33 @@
 import { FC } from 'react';
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
+import useGetCurrentPortfolioValue from 'queries/futures/useGetCurrentPortfolioValue';
+import { FuturesMarket } from 'queries/futures/types';
+import { Synths } from 'constants/currency';
+import Currency from 'components/Currency';
+import { wei } from '@synthetixio/wei';
 
-const PortfolioChart: FC = () => {
+type PortfolioChartProps = {
+	futuresMarkets: FuturesMarket[]
+};
+
+const PortfolioChart: FC<PortfolioChartProps> = ({ futuresMarkets }: PortfolioChartProps) => {
 	const { t } = useTranslation();
 
+	const markets = futuresMarkets.map((market: FuturesMarket) => {return market.asset})
+	const portfolioValueQuery = useGetCurrentPortfolioValue(markets);
+	const portfolioValue = portfolioValueQuery?.data ?? null;
+
 	return (
-		<Chart />
+		<Chart>
+			<PortfolioTitle>Futures Portfolio Value</PortfolioTitle>
```

Comment:
I think this says "Portfolio Value" on the designs, but I guess this is specific for a reason, right?

Reply:
Yes, this is intentional. I think eventually this will say "Portfolio Value" but for now this only includes futures. I think eventually it will cover shorts and spot assets, so want to be more specific for now.